openedx/core/djangoapps/plugins/docs/decisions/0004-plugin-slots.rst

```diff
@@ -0,0 +1,103 @@
+Plugin Slots
+------------
+
+Status
+======
+Draft
+
+Context
+=======
+edx-platform contains a plugin system (https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/plugins)
+which allows new Django apps to be installed inside the LMS and Studio without
+requiring the LMS/Studio to know about them. This is what enables us to move to
+a small and extensible core. While it's possible to extend the content of pages
+rendered by the platform using templates, via certain extension points that allow
+injecting content into 'head-extra', 'body-initial' etc slots in the base template,
+it isn't possible for plugins to inject content at all.
+
+Decisions
+=========
+We have added the ability for plugins to render content into existing pages. To
+support this, we have decided:
+
+* A template can how declare slots into which a plugin can inject content, by
```

Comment:
```suggestion
* A template can now declare slots into which a plugin can inject content, by
```

Reply:
If the new direction seems OK, I will update the docs to match.